smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/SymbolVisitor.java

```diff
@@ -383,14 +390,20 @@ private String flattenShapeName(ToShapeId id) {
     }
 
     private Symbol.Builder createSymbolBuilder(Shape shape, String typeName) {
-        return Symbol.builder().putProperty("shape", shape).name(typeName);
+        return Symbol.builder()
+                .putProperty("shape", shape)
+                .putProperty("traceFileNamespace", moduleNameDelegator.formatModuleName(shape, null))
```

Comment:
These calls to the moduleNameDelegator are causing issues for the model chunking. This needs address before the PR can be merged. Testing with regenerating an existing client should be performed to assure that the model chunking isn't changed unnecessarily for an existing client.